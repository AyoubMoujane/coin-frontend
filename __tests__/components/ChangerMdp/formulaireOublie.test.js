import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { View, Text, TextInput, Button, StyleSheet } from "react-native";


configure({ adapter: new Adapter() });

let findElement = function (tree, element) {
  console.warn(tree)
  return true
}

import FormulaireOublie from '../../../components/ChangerMdp/formulaireOublie'

describe('<formulaireOublie />', () => {
  it("FormulaireOublie renders properly", () => {
    const tree = renderer.create(<FormulaireOublie />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("includes 2 Text labels", () => {
    const wrapper = shallow(<FormulaireOublie />);
    expect(wrapper.find("Text").length).toEqual(2);

  });


});