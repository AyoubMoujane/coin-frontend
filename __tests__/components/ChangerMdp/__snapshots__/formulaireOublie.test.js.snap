// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<formulaireOublie /> FormulaireOublie renders properly 1`] = `
<View
  style={Object {}}
>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 30,
        "marginBottom": 20,
        "paddingTop": "40%",
        "textAlign": "center",
      }
    }
  >
    Entrez votre identifiant
  </Text>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Identifiant"
    rejectResponderTermination={true}
    style={
      Object {
        "alignSelf": "center",
        "borderColor": "grey",
        "borderRadius": 8,
        "borderWidth": 2,
        "height": 30,
        "marginBottom": 10,
        "paddingHorizontal": 10,
        "width": 200,
      }
    }
    underlineColorAndroid="transparent"
    value="identifiant"
  />
  <View
    style={
      Object {
        "alignItems": "center",
      }
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={Object {}}
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          disabled={false}
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          Envoyer
        </Text>
      </View>
    </View>
  </View>
  <Text
    style={
      Object {
        "color": "red",
        "textAlign": "center",
      }
    }
  >
    
  </Text>
</View>
`;
